configs/cfbl_defconfig

```diff
@@ -0,0 +1,5 @@
+CONFIG_MOTORS_ESC_PROTOCOL_DSHOT=y
```

Comment:
the configfile is called cfbl_defconfig eventhough the platform calls it cf21b. Should we try to uniform it somehow?